apps/web/src/components/milestone-logs/MilestoneTable.tsx

```diff
@@ -103,7 +103,7 @@ export const MilestoneTable = ({ category }: MilestoneTableProps) => {
       }
     }
 
-    if (start === end) return '-';
+    if (start === end) return '0 day';
```

Comment:
Think the correct way would be '0 days'

Reply:
It's shame that there is an error in the change of just five characters.

Reply:
happens to the best of us! :P